zcash_client_sqlite/src/lib.rs

```diff
@@ -43,7 +43,7 @@ use zcash_primitives::{
     consensus::{self, BlockHeight},
     legacy::TransparentAddress,
     memo::{Memo, MemoBytes},
-    sapling::{self, Nullifier},
+    sapling::{self},
```

Comment:
Is this `rustfmt`ted?

Reply:
For some reason rustfmt doesn't collapse this to just `sapling`. This code is likely to change further, so I'm going to leave it like this for now.